artio-codecs/src/test/java/uk/co/real_logic/artio/dictionary/generation/EncoderGeneratorTest.java

```diff
@@ -1029,4 +1075,21 @@ private void setDataFieldLength(final Encoder encoder) throws Exception
     {
         setInt(encoder, "dataFieldLength", 3);
     }
+
+    private static void assertThrows(final ThrowingRunnable runnable, final Class<? extends Exception> thrown)
+    {
```

Comment:
Maybe I'm misreading this, but it looks like this assert would pass if no exception is thrown and only fail if an exception of the throw type is thrown?

Wouldn't it be better to use [Junit's built in support for this](https://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html)

Reply:
Yes, I forgot about that functionality; we use JUnit5, which has a different way for asserting on exceptions and could only remember the @Test(Exception.class) way.

I've fixed this up